### PR TITLE
Update segmentation_pipeline.py

### PR DESCRIPTION
**Pull request by 9527-csroad**
GPU inference #27

### CODE REVIEW

**Review by Szuumii — APPROVED** [review]
Good stuff! 
But wait for @wiktorlazarski approve as well 😅

**Review by wiktorlazarski — CHANGES_REQUESTED** [review]
@9527-csroad Great first contribution 🚀! Thanks for all the changes. I'm leaving a one tiny request to be fixed rather easily. 

Another thing I'd like to ask you (iff you have time) - would you consider updating the `README.md` file with model's time benchmarking you performed to solve that issue https://github.com/wiktorlazarski/head-segmentation/issues/27? I think that will be an interesting peace of information for future users/contributors.

File: head_segmentation/segmentation_pipeline.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def __init__(
         self,
         model_path: str = C.HEAD_SEGMENTATION_MODEL_PATH,
         model_url: str = C.HEAD_SEGMENTATION_MODEL_URL,
+        device = torch.device('cpu')
```
[the remark this answers is not in the document]
Comment (9527-csroad) [reply head_segmentation/segmentation_pipeline.py:19]: Thanks for your wonderful project. I have updated the README.md file, and you can revise it as you want. This is my first time to PR, I'm not sure if there are any problems.

[the remark this answers is not in the document]
Comment (9527-csroad) [reply head_segmentation/segmentation_pipeline.py]: I'm sorry my PR is mess, in this process, I have learned some PR tricks and thanks for your project again!

File: head_segmentation/_version.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # __version__ = "1.0.0"  # First version
-__version__ = "1.1.0"  # Segmentation model downloaded from GDrive
+# __version__ = "1.1.0"  # Segmentation model downloaded from GDrive
+__version__ = "1.3.0"  # add gpu inference
```
Comment (wiktorlazarski) [inline head_segmentation/_version.py:3]: I'd like to move changelog from the `_version.py` file to release notes. Can I ask you for a favour: would you remove those commented `__version__`'s tags. I'm looking for having a file which looks something like (contains only such single line as it's content):
```suggestion
__version__ = "1.3.0"
```

**Review by wiktorlazarski — APPROVED** [review]
Thanks for all updates and you contribution. Huge shoutout for updating the `README.md` file. I left one last minor comment/request but other than that everything else looks rock solid 🔥.

Congratulations on your first contribution to the project 🎉! If you want to contribute and learn through it 😉 more lemme know I have a few things in my mind that may be implemented in the repo to improve project but also don't hesitate to implement your ideas - that's always welcome 🤗.

File: head_segmentation/_version.py (hunk 2 above)
Comment (9527-csroad) [reply head_segmentation/_version.py]: Happy to do something for this good project 🚀. If there are any problems, contact me and I will do it.

Comment (wiktorlazarski) [reply head_segmentation/_version.py]: Thanks @9527-csroad 🙌. Lemme collect my thoughts over a weekend and I'll spawn a bunch of issues that may improve the project.

File: head_segmentation/segmentation_pipeline.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (karaposu) [reply head_segmentation/segmentation_pipeline.py]: @9527-csroad thanks for update mate. I am also interested in this project. Would you mind sharing ur discord for a chat?

**Pull request merged**